Modules/CalcOffence-3_0.lua

```diff
@@ -1112,6 +1113,12 @@ function calcs.offence(env, actor, activeSkill)
 						local resist = 0
 						local pen = 0
 						local takenInc = enemyDB:Sum("INC", cfg, "DamageTaken", damageType.."DamageTaken")
+						if (isAttack and enemyDB:Flag(nil, "Condition:Intimidated")) then
+							takenInc = takenInc + 10
+						end
+						if (isSpell and enemyDB:Flag(nil, "Condition:Unnerved")) then
+							takenInc = takenInc + 10
+						end
```

Comment:
Instead of doing custom tooling here - could we figure out a system to differentiate increased taken with spell damage and increased taken with attack damage  which doesnt explicitely reference Intimidate and Unnerved?

Reply:
Is there an existing way to link a condition flag to a specific damage mod?

Reply:
from config tab
		enemyModList:NewMod("DamageTaken", "INC", 10, "Unnerve", ModFlag.Spell)
though I was under the impression we dont implement these kinds of things as plays can just manually check them in the config tab